frontend/scripts/react-components/nodes-panel/nodes-panel.reducer.js

```diff
@@ -266,14 +290,17 @@ const nodesPanelReducer = {
     const { activeItem } = action.payload;
     if (moduleOptions.hasMultipleSelection) {
       return immer(state, draft => {
+        let isANewTab = false;
         if (moduleOptions.hasTabs) {
           draft[name].activeTab = state[name].tabs.find(tab => tab.name === activeItem.nodeType).id;
 
```

Comment:
There is a crash here.
Get into the menu and delete one of the municipalities with the sentence selector:

```redux-logger.js?d665:1 Uncaught TypeError: Cannot read property 'id' of undefined
    at eval (nodes-panel.reducer.js?cebd:295)
    at Immer.produce (immer.module.js?57f8:787)
    at eval (nodes-panel.reducer.js?cebd:292)
    at reducer (createReducer.js?99a3:6)
    at combination (redux.js?00d8:458)
    at p (<anonymous>:1:36402)
    at v (<anonymous>:1:36684)
    at <anonymous>:1:40069
    at Object.dispatch (redux.js?00d8:212)
    at e (<anonymous>:1:40553)```